keystrokes/src/components/Auth/home.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import { auth } from '../../config/constants';
+
+class Home extends Component {
+	logout = e => {
```

Comment:
nice! Just want to make sure you understand the different ways to bind

`functionName = params => {}`
`this.functionName = this.functionName.bind(this)`
`onClick={() => {this.functionName}}`